app/controllers/application_controller.rb

```diff
@@ -4,7 +4,7 @@ class ApplicationController < ActionController::Base
   helper_method :current_admin, :current_user
 
   def current_user
-    @current_user ||= RegisteredUser.find(session[:user_id]) if session[:user_id]
+    @current_user ||= RegisteredUser.find(session[:registered_user_id]) if session[:registered_user_id]
```

Comment:
are we saving anything else in the session as far as user_id's go? could we just put the `user_id` in the session rather than `registered_user_id`?